src/test/java/com/marklogic/kafka/connect/source/SerializedConstraintInjectionTest.java

```diff
@@ -0,0 +1,68 @@
+package com.marklogic.kafka.connect.source;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.Map;
+
+class SerializedConstraintInjectionTest extends AbstractIntegrationSourceTest {
+    private static final String constraintColumn = "lucky_number";
+    private static final String constraintValue = "52";
+
+    private final SerializedConstraintColumnHandler serializedConstraintColumnHandler = new SerializedConstraintColumnHandler(null);
+
+    @Test
+    void testAccessorOnlyQuery() {
+        String originalQuery = "{\"$optic\":{\"ns\":\"op\", \"fn\":\"operators\", \"args\":[{\"ns\":\"op\", \"fn\":\"from-view\", \"args\":[\"Medical\", \"Authors\"]}]}}";
```

Comment:
These strings of JSON are pretty difficult to read and understand. It would help if they're pretty-printed... and I think that will be true in production too - i.e. if we ever want to log something that includes the serialized plan, it'll be far easier to understand if it's pretty-printed. 

I think each of these tests should do what the test below this one does - pretty-print the input and toss it into a String (or what may be better is to toss it into a file under src/test/resources) so that a reader of the test can easily understand what's in the original plan and what's in the expected plan. And then the Serialized implementation should be changed to call "toPrettyString" on the ObjectNode it builds up. There's a very small cost to doing toPrettyString instead of toString that will be dwarfed by the amount of time spent actually getting data back. 

It may be good to have one test where the input is a single line of JSON, which will likely be the case when using a properties file, just to ensure that works. But the expected output would still be pretty-printed.